### PR TITLE
[CI][XPU] Skip fork-dependent logits processor test

### PR DESCRIPTION
**Pull request by zhenwei-intel**
## Purpose
XPU does not support fork, causing the CI to fail, so this PR skips the fork-dependent test.

## Test Result

---
<details>
<summary> Essential Elements of an Effective PR Description Checklist </summary>

- [ ] The purpose of the PR, such as "Fix some issue (link existing issues this PR will resolve)".
- [ ] The test plan, such as providing test command.
- [ ] The test results, such as pasting the results comparison before and after, or e2e results
- [ ] (Optional) The necessary documentation update, such as updating `supported_models.md` and `examples` for a new model.
</details>

### CODE REVIEW

**Comment by zhenwei-intel** [comment]
@jikunshang @zxd1997066 , PTAL

**Review by jikunshang — APPROVED** [review]
